### ISSUE TITLE
GCP Failover for Alias Address fails when no forwarding rules configured

### ISSUE THREAD

**Issue by f5-applebaum** [F5Networks/f5-cloud-failover-extension#74]
## Description

GCP Failover fails with:
 "Cannot read property 'IPAddress' of undefined TypeError: Cannot read property 'IPAddress' of undefined"
when there is alias addresses but no Forwarding Rules:

```
Thu, 17 Jun 2021 17:24:15 GMT - info: [f5-cloud-failover] Performing Failover - discovery
Thu, 17 Jun 2021 17:24:16 GMT - severe: [f5-cloud-failover] error in _getFailoverDiscovery: TypeError: Cannot read property 'IPAddress' of undefined
Thu, 17 Jun 2021 17:24:16 GMT - severe: [f5-cloud-failover] Cannot read property 'IPAddress' of undefined TypeError: Cannot read property 'IPAddress' of undefined
    at fwdRules.forEach (/var/config/rest/iapps/f5-cloud-failover/nodejs/providers/gcp/cloud.js:872:71)
    at Array.forEach (<anonymous>)
    at Promise.all.then (/var/config/rest/iapps/f5-cloud-failover/nodejs/providers/gcp/cloud.js:869:31)
    at tryCatcher (/usr/share/rest/node/node_modules/bluebird/js/release/util.js:16:23)    at Promise._settlePromiseFromHandler 
(/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:638:18)
    at PromiseArray._resolve (/usr/share/rest/node/node_modules/bluebird/js/release/promise_array.js:126:19)
    at PromiseArray._promiseFulfilled (/usr/share/rest/node/node_modules/bluebird/js/release/promise_array.js:144:14)
    at Promise._settlePromise (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:574:26)
    at Promise._settlePromise0 (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/share/rest/node/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/usr/share/rest/node/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
Thu, 17 Jun 2021 17:24:24 GMT - finest: socket 204 closed
```

## Expected Behavior 

You should be able to just perform Alias Address Failover. You should not need Forwarding Rules configured to do Alias Address Failover. The two use cases should not be coupled.  

Tried with explict config with no forwarding rules defined as well:
```
  "failoverAddresses": {
    "enabled": true,
    "addressGroupDefinitions": [
      {
        "type": "aliasAddress",
        "scopingAddress": "10.0.12.112/28"
      }
    ]
  },
```
but same result. 

## Environment information
For bugs, enter the following information:
- Cloud Failover Extension Version: 1.8.0
- BIG-IP version: 15
- Cloud provider: GCP

## Severity Level

Severity: <Fill in level: 1 through 5>
**Severity 2 (High)** : Defect is causing major obstruction of system operations.

**Comment by shyawnkarim** [comment]
This is being tracked internally with ID AUTOSDK-548.

**Comment by shyawnkarim** [comment]
Closing.

This issue was addressed with [Release 1.10](https://github.com/F5Networks/f5-cloud-failover-extension/releases/tag/v1.10.0).
